NFT-Marketplace-Part-2.md

```diff
@@ -147,89 +149,91 @@ There are three main files we will be touching in this project, and should take
 If we open that file, it should look something like this:
 
 ```yaml
-specVersion: 0.0.2
+specVersion: 0.0.4
 schema:
   file: ./schema.graphql
 dataSources:
   - kind: ethereum
     name: NFTMarketplace
     network: celo-alfajores
     source:
-      address: "0x9014DD98Cd14B26c76069356247cE6d762018220"
+      address: "0x627f152f97d431B844604B4421313CA979712006"
       abi: NFTMarketplace
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.5
       language: wasm/assemblyscript
       entities:
-        - ListingCanceled
+        - ListingCancelled
```

Comment:
This is an American vs British spelling thing. I'm fine changing it, but this also requires a change in the smart contract in part 1.